### ISSUE TITLE
Create a minimal build of Bootstrap v3

### ISSUE THREAD

**Issue by fabd** [fabd/kanji-koohii#46]
There are loads of things I don't really need from Bootstrap, and I don't want to use in the future because it create more dependencies.

Currently the website mainly uses: **reset, grid, base typography, forms, buttons**.

The first four are basically the most vital elements I benefited from using Bootstrap in the mobile / responsive update and not much else is needed for reasons explained below.

**Solution**

As of writing I don't know if it's worth building Bootstrap vs just making a custom package from their site since I don't plan on using it more while transitioning to Vue JS components.

**Random blabbering**

Later I may switch to Bootstrap v4. I have to look at Bulma as well and the [Vue-Blu](https://chenz24.github.io/vue-blu/#/) component kit.

It's hard at this time to find a solid Vue Js 2  component kit without too much dependencies. **The Bootstrap based ones are quite bad** as far as I have seen, which is why I don't want to use more of Bootstrap styles.

Besides in general the styles are quite simple, such as the Panel, in which case it's better to have standalone styles in the vue files, without the CSS framework dependencies. I couldn't even use the Bootstrap nav for the navigation in desktop view, because it didn't quite behave the way I want to AND the way it handles the mobile navigation was too restricting. Besides we now have a mobile nav made from an instanced VueJS component, so the Bootstrap nav was of no use.

Since Vue 2 is still relatively new it means I can't use a single Vue 2 component kit because they all have their quirks and problems. So I hand pick some simple components here and there that I can build on, and that further emphasizes why we don't want dependency on a single CSS framework.

Basically the key to manage this site the past ten years, is to always consider what dependencies I have. Keep the css organized in a "component" fashion. All the styles applying to a given area or reusable component are lumped together. When I refactor, it's much easier to drop big chunks of CSS because they don't depend on a complex cascade. With `.vue` files it's even better as styles can be kept in the Vue file, ensuring we can replace plug in components, or replace them as needed without having all kind of side effects.

**Comment by fabd** [comment]
Done.

**Landing Page**
Main stylesheet : 125 kb => 45.1 (before gzip)

Total page load: 15 requests, **349 KB**

**Signed in, home page** (adds elgacy YUI 2 bundle dependency)
Main stylesheet : 153 kb => 72 kb  (before gzip)

Total page load: 11 requests, **208 KB**
